updates-notifier@zamszowy: hide action menus if associated command is empty

yrtimiD opened the pull request:
Allow hiding menu commands via settings.

<img width="696" height="241" alt="image" src="https://github.com/user-attachments/assets/38de65e8-6779-4ca0-9017-28bfce289652" />

<img width="686" height="138" alt="image" src="https://github.com/user-attachments/assets/3d1bd33d-f21d-43e5-ace6-9670332e1675" />

<img width="174" height="87" alt="image" src="https://github.com/user-attachments/assets/9ce1fb92-0da6-47c7-81d9-3daead9124fc" />

owner: @zamszowy

zamszowy reviewed (commented):
It would be nice to have `"tooltip"` descriptions in `settings-schema.json` file for `commandUpdate` and `commandUpgrade`, stating that the empty command would disable that entry popup, e.g. "leave empty to disable this popup entry", so that the user would know that there is such an option.

yrtimiD commented:
Totally agree. 
Also, I'm not comfortable with the fact that it's not trivial to reset commands to their default values. Maybe it's better to wrap each command with toggle as you did for "different-levels"? And then it will be obvious how to hide unwanted menus. WDYT?

yrtimiD commented:
@zamszowy , replaced "is empty" logic with explicit settings. 
Slightly refactored menu construction in the _update() function to avoid double checking if menu should be shown (hope you don't mind).
Also renamed "Check for updates" setting to "Check for new updates" to match the actual menu item.

zamszowy commented on updates-notifier@zamszowy/files/updates-notifier@zamszowy/po/it.po:
```diff
@@ -98,9 +98,20 @@ msgstr ""
+#. settings-schema.json->commandUpdate-show->description
+#, fuzzy
+msgid "Show \"Check for new updates\" command"
+msgstr "Cerca nuovi aggiornamenti"
```
```suggestion
msgstr ""
```

zamszowy commented on updates-notifier@zamszowy/files/updates-notifier@zamszowy/po/it.po:
```diff
@@ -98,9 +98,20 @@ msgstr ""
-msgstr "Verifica aggiornamenti:"
+#, fuzzy
+msgid "Check for new updates:"
+msgstr "Cerca nuovi aggiornamenti"
```
```suggestion
msgstr "Cerca nuovi aggiornamenti:"
```

zamszowy commented on updates-notifier@zamszowy/files/updates-notifier@zamszowy/po/it.po:
```diff
@@ -98,9 +98,20 @@ msgstr ""
+#. settings-schema.json->commandUpgrade-show->description
+#, fuzzy
+msgid "Show \"Upgrade packages\" command"
+msgstr "Aggiorna pacchetti:"
```
```suggestion
msgstr ""
```

zamszowy commented on updates-notifier@zamszowy/files/updates-notifier@zamszowy/po/nl.po:
It seems that the translation got misaligned after introducing new strings. Could you please correct this file in the same manner as I proposed for `it.po`?

yrtimiD replied:
hold on... not yet...

yrtimiD replied:
ready, please take a look.
since "Check for new updates" appeared twice, I copied translated value.

AlexOnFOSS commented:
@zamszowy Sorry for writing in here. I have found a bug in updater-notifiert in debian13 cinamon desktop. How can I post you the bug and the solution to it? I am very unfamiliar with github.
